### PR TITLE
Update izone documentation for new control zone sensors

### PR DESCRIPTION
**Pull request by sgryphon**
## Proposed change

Documentation to correspond with core PR  43984, which adds information about the current control zone (when there are multiple zones).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/43984
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
This PR is targeting an incorrect branch. This PR describes a new feature and thus does not apply the the `current` documentation, but to the `next` documentation release.

Please change the target branch to `next` and git rebase your PR is needed.

Could you also update the title of the PR to be more descriptive of what it does? The core PR number itself doesn't ring a bell when looking at the list of PRs.

Thanks 👍

**Comment by sgryphon** [comment]
> Please change the target branch to `next` and git rebase your PR is needed.
> Could you also update the title of the PR to be more descriptive of what it does?

Rebased and target changed.

Name changed to reflect content

**Comment by frenck** [comment]
Right, so the builds need to pass 😉

File: source/_integrations/izone.markdown [hunk 1]
```diff
@@ -54,6 +54,50 @@ Unit modes off, heat, cool, dry, and fan only are supported. For units fitted wi
...
+The climate controller then selects the zone that is furthest away from the target and feeds the current temperature and
+target temperature into the air conditioner unit, closing any other zones that have already reached their target.
+
+In this mode the current control zone that has been selected is reported, as is the readonly target temperature for that 
+zone (read only, set the value via the individual zones). The current temperature will also be that of the control
+zone.
+
+You can add configure to read these values into sensors, along with the supply temperature (use the ID of your unit):
+
+```
```
Comment (klaasnicolaas) [inline source/_integrations/izone.markdown]:
```suggestion
```yaml
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/izone.markdown (hunk 1 above)
Comment (sgryphon) [reply source/_integrations/izone.markdown]: Okay, I worked out how to escape out the templating. (Found it in bayesian.markdown, use {% raw %}).

Build now passes.

**Comment by sgryphon** [comment]
The PR code change, that this is the documentation for, is now merged: https://github.com/home-assistant/core/pull/43984

File: source/_integrations/izone.markdown [hunk 2]
```diff
@@ -54,6 +54,54 @@ Unit modes off, heat, cool, dry, and fan only are supported. For units fitted wi
 
 Zones have three modes available, closed, open, and auto. These are mapped to Home Assistant modes off, fan only, and auto, respectively. Only the auto mode supports setting the temperature.
 
+## Control zone (climate control mode)
+
+With multiple climate controlled zones you can't set the target temperature of the control, but set the target temperature
+for each individual zone.
```
Comment (frenck) [inline source/_integrations/izone.markdown]:
```suggestion
With multiple climate-controlled zones, you can't set the target temperature of the control but set the target temperature
for each individual zone.
```

File: source/_integrations/izone.markdown [hunk 3]
```diff
@@ -54,6 +54,54 @@ Unit modes off, heat, cool, dry, and fan only are supported. For units fitted wi
...
+
+The climate controller then selects the zone that is furthest away from the target and feeds the current temperature and
+target temperature into the air conditioner unit, closing any other zones that have already reached their target.
+
+In this mode the current control zone that has been selected is reported, as is the readonly target temperature for that 
+zone (read only, set the value via the individual zones). The current temperature will also be that of the control
+zone.
```
Comment (frenck) [inline source/_integrations/izone.markdown]:
```suggestion
In this mode the current control zone that has been selected is reported, as is the read-only target temperature for that 
zone (read-only, set the value via the individual zones). The current temperature will also be that of the control
zone.
```

File: source/_integrations/izone.markdown [hunk 4]
```diff
@@ -54,6 +54,54 @@ Unit modes off, heat, cool, dry, and fan only are supported. For units fitted wi
...
+        friendly_name: 'Control zone'
+        value_template: '{{ state_attr("climate.izone_controller_0000XXXXX","control_zone_name") }}'
+      control_zone_target:
+        friendly_name: 'Target temperature'
+        value_template: '{{ state_attr("climate.izone_controller_0000XXXXX","control_zone_setpoint") }}'
+        unit_of_measurement: '°C'
+      temperature_supply:
+        friendly_name: 'Supply temperature'
+        value_template: '{{ state_attr("climate.izone_controller_0000XXXXX","supply_temperature") }}'
+        unit_of_measurement: '°C'
```
Comment (frenck) [inline source/_integrations/izone.markdown]:
```suggestion
        friendly_name: "Control zone"
        value_template: "{{ state_attr('climate.izone_controller_0000XXXXX','control_zone_name') }}"
      control_zone_target:
        friendly_name: "Target temperature"
        value_template: "{{ state_attr('climate.izone_controller_0000XXXXX','control_zone_setpoint') }}"
        unit_of_measurement: "°C" 
      temperature_supply:
        friendly_name: "Supply temperature"
        value_template: "{{ state_attr('climate.izone_controller_0000XXXXX','supply_temperature') }}"
        unit_of_measurement: "°C"
```

File: source/_integrations/izone.markdown [hunk 5]
```diff
@@ -54,6 +54,54 @@ Unit modes off, heat, cool, dry, and fan only are supported. For units fitted wi
...
+In this mode the current control zone that has been selected is reported, as is the readonly target temperature for that 
+zone (read only, set the value via the individual zones). The current temperature will also be that of the control
+zone.
+
+You can add configure to read these values into sensors, along with the supply temperature (use the ID of your unit):
+
+{% raw %}
+
+```yaml
+sensor:
```
Comment (frenck) [inline source/_integrations/izone.markdown:77]: I think it is unclear for the reader where this YAML snippet should go.

File: source/_integrations/izone.markdown [hunk 6]
```diff
@@ -54,6 +54,54 @@ Unit modes off, heat, cool, dry, and fan only are supported. For units fitted wi
...
+        value_template: '{{ state_attr("climate.izone_controller_0000XXXXX","supply_temperature") }}'
+        unit_of_measurement: '°C'
+```
+
+{% endraw %}
+
+And then graph them on a dashboard, along with the standard values such as the current temperature:
+
+```yaml
+entities:
```
Comment (frenck) [inline source/_integrations/izone.markdown:100]: I think it is unclear to the reader where this YAML snippet should go.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/izone.markdown (hunk 5 above)
Comment (sgryphon) [reply source/_integrations/izone.markdown:77]: Additional comments added to explain where it goes.

File: source/_integrations/izone.markdown (hunk 6 above)
Comment (sgryphon) [reply source/_integrations/izone.markdown:100]: Additional comments added to explain where it goes.

**Comment by sgryphon** [comment]
Thanks for the feedback @frenck . I have made the suggested edits.

**Review by frenck — APPROVED** [review]
Thanks, @sgryphon 👍

**Pull request merged**